src/common/MuxConfig.h

```diff
@@ -316,6 +316,24 @@ class MuxConfig
      */
     inline bool getIfEnableDefaultRouteFeature() {return mEnableDefaultRouteFeature;};
 
+    /**
+     * @method getUseKnownMacActiveActive
+     * 
+     * @brief check if use known mac to probe for active-active ports
+     * 
+     * @return true to use known mac to probe for active-active ports
+     */
+    inline bool getUseKnownMacActiveActive() { return mUseKnownMacActiveActive; };
```

Comment:
Better named as `getIfUseKnownMacActiveActive` as return value is a `bool`?

Reply:
Updated